test/runtests.jl

```diff
@@ -1,51 +1,93 @@
 using GigaSOM
 using Test
 using Random
+using Distributed
 
 #fix the seed
 Random.seed!(1)
 
 include("io.jl")
 
-#BATCH & PARALLEL
+#test cleannames
+@testset "cleannames" begin
```

Comment:
May you put these tests into external file(s)?

Reply:
ok, like the `parallel.jl`?

Reply:
done